### ISSUE TITLE
co-authors can't edit their posts

### ISSUE THREAD

**Issue by blauertee** [Automattic/Co-Authors-Plus#644]
This might be easy to solve but I've no idea how to debug what's happening here.

The plugin seems to work fine but co-authors can't edit the posts they're assigned to. Which capabilities do users need to edit post's of which they're coauthors?

capabilities the coauthor has (insufficient??):
`create_posts` `edit_published_pages` `edit_pages edit_posts` `edit_published_pages` `edit_published_posts` `list_users` `manage_categories` `moderate_comments` `publish_pages` `publish_posts` `read` `read_private_pages` `read_private_posts`

I'm also using additional custom roles and not the default wordpress Subscriber, Contributor, ... , Admin roles. Could that be a problem?

Thanks!

**Comment by bcd123** [comment]
I have the same (or a similar?) problem. 

WordPress 5.0.3
Co-Authors-Plus Version 3.3.1

*Authors can edit their posts
*Co-authors (who are not first authors) cannot edit posts (page does not load, 403)

If I add the "edit_others_posts" ability to the co-author in question then they can edit the post. But they can also now edit all of the posts.

**Comment by bcd123** [comment]
I see this error in the console on chrome:

api-fetch.min.js?ver=2.2.7:formatted:358 GET https://MYSITE.com/wp-json/wp/v2/posts/24815?context=edit&_locale=user 403 (Forbidden)

When I try that url manually in the browser I get:

{"code":"rest_forbidden_context","message":"Sorry, you are not allowed to edit this post.","data":{"status":401}}

I have confirmed that the user is listed as a co-author on the post.

**Comment by blauertee** [comment]
Okay interesting I'm also on WP 5.0.3 
Are you using other plugins? 

I've got these in use:
```
Co-Authors Plus, Version 3.3.1 
Database Browser, Version 1.4 
Extend Media Upload, Version 1.09
Google XML Sitemaps, Version 4.1.0 	
Lightbox Galerie, Version 0.9.1	
Organize Media Library by Folders, Version 6.44	
pTypeConverter, Version 0.2.8.1	
User Role Editor, Version 4.49
WP Emmet, Version 0.3.4
```
I can't really reproduce the error message, what have you done to get it, does the edit button exist for coauthors? On my side they don't even get the button.

But if a coauthor simply opens the uri `http://MYSITE.com/wp-admin/post.php?post=POST_ID&action=edit` I get the same text displayed on the screen ("Sorry, you are not allowed to edit this post.")

**Comment by jtomeck** [comment]
@bcd123 and I are also using User Role Editor, but I have disabled it (as well as all other plugins) on a development instance of our site to test and we are still experiencing the same issue.

I was hoping having it in a development space with PHP error reporting enabled would show some kind of error, but I'm still getting a blank page, which means the actual PHP is not the culprit.

**To reproduce:**
1. Create a post
2. Create test user account
3. Add that test user as a co-author
4. Log in as test user and try to edit post
5. Test user sees a blank page. Error message can be seen in developer tools console (I'm using chrome)

**Comment by blauertee** [comment]
> 4\. Log in as test user and try to edit post
But there is no edit button right, you enter the edit url manually? If that's the case I guess we've got the same issue.

**Comment by jtomeck** [comment]
Oh interesting. There is an edit link under the post title in the list. See screenshot:

<img width="300" alt="screen shot 2019-02-12 at 12 59 58 pm" src="https://user-images.githubusercontent.com/1560779/52657205-22936c80-2ec6-11e9-8c21-cdffa787d8bd.png">

**Comment by blauertee** [comment]
I only get the Preview button.

**Comment by richa2010** [comment]
https://wordpress-344534-1445342.cloudwaysapps.com/my-dashboard/austcybers/?action=edit&job_id=5345&account=no

Can't access this  page when i login with co author users

**Comment by nielslange** [comment]
@blauertee, @bcd123, @jtomeck @richa2010 This problem sounds like https://github.com/Automattic/Co-Authors-Plus/issues/754, which had been fixed in https://github.com/Automattic/Co-Authors-Plus/pull/757 and released with version 3.4.4. 

I'll close this issue for now, but feel free to reopen it, in case you're still facing problems with this case.
